astropy/utils/shapes.py

```diff
@@ -329,6 +330,8 @@ def __init__(self, shape_a, shape_a_idx, shape_b, shape_b_idx):
 
 
 def check_broadcast(*shapes):
+    warn("check_broadcast() method has been deprecated. You should use numpy.broadcast_shapes() method instead.")
```

Comment:
Please use the `@deprecated` decorator. You can see how we deprecate other things in this code base for example, or other similar PRs. Thanks!

Reply:
Done! Thank you for pointing that out!